send-transaction-service/src/send_transaction_service.rs

```diff
@@ -481,7 +480,6 @@ impl SendTransactionService {
                             let entry = retry_transactions.entry(signature);
                             if let Entry::Vacant(_) = entry {
                                 if retry_len >= config.retry_pool_max_size {
-                                    datapoint_warn!("send_transaction_service-queue-overflow");
```

Comment:
Can you fix this instead of removing, please?

Reply:
Ah, thanks for pointing that out. I guess the main difference is this metric fires on `warn` log level. But given that it hasn't been working properly, hopefully no one is relying on it. Okay with me to remove after all.